tests/conftest.py

```diff
@@ -245,6 +245,20 @@ def excepthook(cls, exc, tb):
     monkeypatch.setattr(sys, "excepthook", excepthook)
 
 
+@pytest.fixture(params=[False, True])
+def try_queue_and_scheduler(request, monkeypatch):
```

Comment:
For some reason, the test that is being decorated with this fixture also needs to specify the `monkeypatch` fixture. Don't know why (otherwise it will silently ignore the request to enable scheduler).